test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -62,12 +62,25 @@ call_log_manager::process_call_log()
 void
 call_log_manager::process_call_log_entry(json call_log_entry)
 {
-    const std::string method_name = call_log_entry["MethodName"].get<std::string>();
+    const std::string method_name = call_log_entry["method_name"].get<std::string>();
     switch (_api_map.at(method_name)) {
     case api_method::wiredtiger_open:
-        /* conn = &connection_simulator::get_connection(); */
+        _conn = &connection_simulator::get_connection();
         break;
     case api_method::open_session:
+        if (_conn == nullptr){
+            std::cerr << "Could not open session, connection does not exist." << std::endl;
+            break;
+        }
+
+        session_simulator *session = _conn->open_session();
+        /* 
+        * Insert this session into the mapping between the simulator session object and the
+        * wiredtiger session object. 
+        */
+       const std::string session_id = call_log_entry["session_id"].get<std::string>();
+        _session_map.insert(std::pair<std::string, session_simulator *>(
```

Comment:
I think we should check whether the session already exists or not (In case someone manually changes the call log file) and exit if it does.

Reply:
Fair enough.

Reply:
Fixed.